deriva/core/ermrest_model.py

```diff
@@ -1,15 +1,16 @@
 
 from __future__ import annotations
 
+import base64
```

Comment:
Hmmm... this must be what pycharm meant by "optimizing" imports on commit. I'm gonna not do that in the future. If this is annoying I can revert this part but its just a reordering as far as I see.

Reply:
I think it's ok